referenceframe/model.go

```diff
@@ -237,11 +237,17 @@ func floatsToString(inputs []Input) string {
 func sortTransforms(unsorted map[string]Frame, parentMap map[string]string, start, finish string) ([]Frame, error) {
 	seen := map[string]bool{}
 
-	nextTransform := unsorted[start]
+	nextTransform, ok := unsorted[start]
+	if !ok {
+		return nil, errors.Errorf("frame named '%v' not in the list of transforms", start)
+	}
 	orderedTransforms := []Frame{nextTransform}
 	seen[start] = true
 	for {
-		parent := parentMap[nextTransform.Name()]
+		parent, ok := parentMap[nextTransform.Name()]
+		if !ok {
+			return nil, errors.Errorf("frame named '%v' has no parent specified", nextTransform.Name())
```

Comment:
This isn't the problem here.  The issue is that the frame it does have as a parent isn't in the map of parents.  Can you change the error message to reflect this?

Reply:
I have this now:

```go
// NewParentFrameNotInMapOfParentsError returns an error indicating that a parent from of the given name
// is missing from the provided map of parents.
func NewParentFrameNotInMapOfParentsError(parentFrameName string) error {
	return errors.Errorf("parent frame named '%s' not in the map of parents", parentFrameName)
}
```

Does this look good?

Reply:
Yep!